### PR TITLE
Add "Groups" users are belong to into users list

### PR DESCRIPTION
**Pull request by GitSumito**
#2966

### CODE REVIEW

**Comment by GitSumito** [comment]
It seems that the function which I modified are too complex..
It should be fixed.

Does someone have a good idea??

File: redash/cli/users.py [hunk 1]
```diff
@@ -247,3 +247,10 @@ def list(organization=None):
 
         print("Id: {}\nName: {}\nEmail: {}\nOrganization: {}\nActive: {}".format(
             user.id, user.name.encode('utf-8'), user.email, user.org.name, not(user.is_disabled)))
+
+        for group_id in user.group_ids:
+            try:
+                group = models.Group.query.get(group_id)
+                print("Group: {}".format(group.name))
+            except AttributeError:
```
Comment (kravets-levko) [inline redash/cli/users.py]:
```python
groups = models.Group.query.filter(models.Group.id.in_(user.group_ids)).all()
group_ids = [group.id for group in groups]
print("Groups: {}".format(", ".join(group_ids)))
```

Comment (GitSumito) [reply redash/cli/users.py]: But did not work well.

Comment (GitSumito) [reply redash/cli/users.py]: I'd like to display group name, not group id.

Comment (kravets-levko) [reply redash/cli/users.py]:
```python
group_names = [group.name for group in groups]
print("Groups: {}".format(", ".join(group_names)))
```
:-)

**Pull request merged**